spirv_hlsl.cpp

```diff
@@ -2712,13 +2791,18 @@ void CompilerHLSL::emit_buffer_block(const SPIRVariable &var)
 
 		std::string type_name;
 		if (is_user_type_structured(var.self))
-			type_name = join(is_readonly ? "" : is_interlocked ? "RasterizerOrdered" : "RW", "StructuredBuffer<", to_structuredbuffer_subtype_name(type), ">");
+			type_name = join(is_readonly    ? "" :
```

Comment:
Is this a functional change? Looks like just restyling, but hard to tell at a glance. If it's not needed, please don't reflow code randomly.

Reply:
I tried to only run clangformat in the functions I touched, figuring that was a good balance.

I'll make the formatting a bit more conservative.